ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -237,11 +237,29 @@ done
      	  $EXTRA_ARGS \
 	  -nographic) > "$workFolder/${OS}.${ARCHITECTURE}.startlog" 2>&1 &
 
-	echo "Machine is booting; logging console to $workFolder/${OS}.${ARCHITECTURE}.startlog Please be patient"
-	sleep 180
-	tail "$workFolder/${OS}.${ARCHITECTURE}.startlog" | sed 's/^/CONSOLE > /g'
-	echo "Machine has started, unless the above log shows otherwise ..."
+	echo "Machine is booting; logging console to $workFolder/${OS}.${ARCHITECTURE}.startlog"
+	echo "Please be patient"
 
+	SECONDS=0
+	while [ true ];
+	do
+		if tail "$workFolder/${OS}.${ARCHITECTURE}.startlog" | grep -q login; then
+			echo "VM Booted after $SECONDS seconds"
+			break;
+		fi
+		if [ $SECONDS -gt 180 ]; then
```

Comment:
Since we're able to trap it as quick as possible and we're not waiting for the full amount of time I'd be tempted to increase this
```suggestion
		if [ $SECONDS -gt 300 ]; then
```

Reply:
Fixed with latest push :-)